policy_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2013-2015 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"testing"
+
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcutil"
+)
+
+// TestCalcMinRequiredTxRelayFee tests the calcMinRequiredTxRelayFee API.
+func TestCalcMinRequiredTxRelayFee(t *testing.T) {
+	tests := []struct {
+		size     int64          // Transaction size in bytes.
```

Comment:
Please give the tests a `name  string` field and include it in the error output.  I know there are several tests that don't do that currently, but we've been trying to move that direction since it's much harder to figure out which test failed when only given a number.  Granted it's not too difficult when there are only a few tests, but I'd like to be consistent.